include/nicegraf-util.h

```diff
@@ -93,6 +93,19 @@ static inline size_t ngf_util_align_size(size_t value, size_t alignment) {
     return value + (m > 0 ? (alignment - m) : 0u);
 }
 
+/**
+ * \ingroup ngf_util
+ *
+ * Returns the largest sample count in a sample count bitmap
+ */
+static inline ngf_sample_count ngf_util_get_highest_sample_count(size_t counts_bitmap) {
```

Comment:
I would actually prefer to have explicit separate fields in the `ngf_device_capabilities` struct indicating the max supported sample count, i.e.:  `max_supported_framebuffer_color_sample_count`, etc.

You can still use this function to compute them, just move it to the internal lib (`source/ngf-common/internal.c`).